CMakeLists.txt

```diff
@@ -189,11 +188,11 @@ IF(ENABLE_TLS)
         SET_PROPERTY(TARGET valkey_tls PROPERTY LINK_FLAGS "-Wl,-undefined -Wl,dynamic_lookup")
     ENDIF()
 
-    SET_TARGET_PROPERTIES(valkey_tls
-        PROPERTIES
-        WINDOWS_EXPORT_ALL_SYMBOLS TRUE
-        SOVERSION "${VERSION_MAJOR}"
-        VERSION "${VERSION}")
+    set_target_properties(valkey_tls PROPERTIES
+      C_VISIBILITY_PRESET hidden
```

Comment:
What's the indentation rule? Is it 2 spaces after an open ( and 4 spaces otherwise?

Reply:
The file is a mix of the older 4 spaces and the newer 2.
All recent changes uses 2 spaces and lowercase for commands.
There is no official guideline, but the cmake-mode.el gives 2.

[Zephyrs](https://docs.zephyrproject.org/latest/contribute/style/cmake.html) guideline is also mentioned sometimes which uses 2.